### ISSUE TITLE
Repair broken CI

### ISSUE THREAD

**Issue by tsaoyu** [tsaoyu/D3HRE#11]
Test passed on my local machine but failed online.
Unable to find the ocean current file that only on my location machine.
Error could be found at:
https://travis-ci.org/tsaoyu/D3HRE/jobs/381725846#L644

Two way to solve this problem:
1. Download the ocean current file (1.3GB) on Travis and change the file directory (no I don't want abuse open-source test service)
2. Using URL + xarray to subset the file online, but it is not possible now due to 

- xarray is not able to deal with `.nc.gz` URL ( https://github.com/pydata/xarray/pull/817)
- the [OSCAR](https://podaac.jpl.nasa.gov/dataset/OSCAR_L4_OC_third-deg) `.nc.gz` file is not actually gzipped, `cat` the file the header is `CDF`

In short term, I would disable the ocean current by default. In a long run,  I would to see `xarray` working with URL.

**Comment by tsaoyu** [comment]
The most recent CI [test](https://travis-ci.org/tsaoyu/D3HRE/jobs/439135505) break down again. 

```
E   AttributeError: 'module' object has no attribute 'AxisError'
```

This is related to an issue of [xarray](https://github.com/pydata/xarray/issues/2483) because they drop the support of Python 3.4 recently. One of their argument is Python 3.4 is already an obsolete version. I would like to support the latest three releases and I will drop the support on Python 3.4 once 3.7 support is there. This also reinforce my idea in preparing a dockerise version of D3HRE once I finish my PhD.

**Comment by tsaoyu** [comment]
Python 3.7 support isn't there yet on Travis. It only support Python version up to `3.7-dev`. <sup>1

1: https://docs.travis-ci.com/user/languages/python/
